packages/block-editor/package.json

```diff
@@ -28,6 +28,7 @@
 		"@wordpress/blocks": "file:../blocks",
 		"@wordpress/components": "file:../components",
 		"@wordpress/compose": "file:../compose",
+		"@wordpress/core-data": "file:../core-data",
```

Comment:
This is not something we should be doing. Adding `core-data` as a dependency to the generic BlockEditor module (WP agnostic).

Reply:
I think we can probably use the BlockListBlock filter to add this in the Editor package instead.

Reply:
Yeah, good idea.

I paused when doing this initially, but then went with it, because block-library does depend on core-data now, but that is not WP agnostic as block-editor is.